labs/coding-102-rest-python-ga/1.md

```diff
@@ -42,31 +40,31 @@ For more information, click the "How to Set Up Your Computer" link at the top of
 * **If you are working on a DevNet Learning Lab computer at a DevNet event**,
     * Open the Git Command window by clicking the *Git CMD* icon on the Task Bar or click the Start button. Then, in the Run bar, enter: `git cmd`.
 * **If you are working from your computer**, on your computer open a command terminal.
-![](/posts/files/coding-102-rest-python-ga/assets/images/create_directory.png)<br/><br/>
+![](assets/images/create_directory.png)<br/><br/>
 * Clone the coding skills sample code from GitHub. Enter the following command. This will copy Cisco DevNet's coding samples directory down to your computer so you may work with it locally.
 ```
 git clone https://github.com/CiscoDevNet/coding-skills-sample-code
 ```
```

Comment:
Ohhh I wonder if we should have them clone dnav3-code like they do in https://github.com/CiscoDevNet/intro-dnac-labs. Yeah, probably need to switch out the code examples... I can help.

Reply:
I can chat with you about this today.

Reply:
Nope, not switching the repo, rather, we'll work on code in the existing code-skills-sample-code repo.